nix/seabug.nix

```diff
@@ -0,0 +1,26 @@
+{ config, lib, pkgs, inputs, ... }:
+
+let
+  cfg = config.seabug;
+in
+{
+  options.seabug = {
+
+    enable = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+    };
+
+  };
+
+  config = lib.mkIf cfg.enable {
+    nft-marketplace-frontend.enable = true;
+
+    nft-marketplace-server = {
+      enable = true;
+      nftStorageKey = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJkaWQ6ZXRocjoweDc5OTNBMjY1NDg5NzE2RmMzMEY2RjFEZjlBOTY0NjM5ZEQzQzViZTAiLCJpc3MiOiJuZnQtc3RvcmFnZSIsImlhdCI6MTY2NTc0MjYxNjA5NywibmFtZSI6InNlYWJ1ZyJ9.2rGfgk3JT2_fBtouWu51dj0jQd3SrnI-NelLq-i5P_U";
```

Comment:
isn't this a secret? in that case it should not be in the store (or the repo). we should use a path to a secret file instead

Reply:
this should not be set by the `seabug` module. it should be set by the nixos configuration/profile that enables the seabug module.

Reply:
Yes it is, for now this is from my personal account (don't mind if someone abuses it). Somewhere I have a TODO saying to manage this using `agenix`. Then I'll put this definition where we define `services.seabug = true`.